### PR TITLE
d4mac, d4win 17.06.1 stable, 17.07.0 edge release

### PR DESCRIPTION
**Pull request by londoncalling**
### What's changed

- added edge release notes for d4mac 17.07.0 
- added edge release notes for d4win 17.07.0
- added stable release notes for d4mac 17.06.1 
- added stable release notes for d4win 17.06.1

### Reviewers

@jeanlaurent @friism @ebriney @djs55 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by mstanleyjones** [comment]
You shouldn't base this on master should you?

**Comment by londoncalling** [comment]
@mstanleyjones I'll wait to merge until the release is out

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -398,6 +412,59 @@ events or unexpected unmounts.
...
+- GUI settings are now stored in `~/Library/Group\ Containers/group.com.docker/settings.json`. `daemon.json` in now a file in `~/.docker/`
+- You can now change the default IP address used by Hyperkit if it collides with your network
+
+**Bug fixes and minor changes**
+
+- Add daemon options validation
+- Diagnose can be cancelled & Improved help information. Fixes [docker/for-mac#1134](https://github.com/docker/for-mac/issues/1134), [docker/for-mac#1474](https://github.com/docker/for-mac/issues/1474)
+- Support paging of Docker Cloud [repositories](/docker-cloud/builds/repos.md) and [organizations](/docker-cloud/orgs.md). Fixes [docker/for-mac#1538](https://github.com/docker/for-mac/issues/1538)
+
+### Docker Community Edition 17.06.1-ce-mac20, 2017-07-18 (edge)
```
Comment (mstanleyjones) [inline docker-for-mac/release-notes.md]: This is duplication of line 22 and I guess it's out of order?

Comment (londoncalling) [inline docker-for-mac/release-notes.md]: @mstanleyjones I think you are seeing the entries for `Stable` and `Edge`

Comment (mstanleyjones) [inline docker-for-mac/release-notes.md]: It's the same build number `mac20`. I don't see how the same build number can be both stable and edge.

**Comment by londoncalling** [comment]
@mstanleyjones this is the way these are shown on stable and edge changelogs, which speaks to the problems we were talking about earlier, so leaving for now and I'll show you the changelogs in the private repo.

**Pull request merged**